modules/ipi-install-creating-an-rhcos-images-cache.adoc

```diff
@@ -128,10 +114,10 @@ registry.centos.org/centos/httpd-24-centos7:latest
 ----
 ifndef::upstream[]
 +
-The above command creates a caching webserver with the name `rhcos_image_cache`, which serves the images for deployment. The first image `${RHCOS_PATH}${RHCOS_QEMU_URI}?sha256=${RHCOS_QEMU_SHA_UNCOMPRESSED}` is the `bootstrapOSImage` and the second image `${RHCOS_PATH}${RHCOS_OPENSTACK_URI}?sha256=${RHCOS_OPENSTACK_SHA_COMPRESSED}` is the `clusterOSImage` in the `install-config.yaml` file.
+The above command creates a caching webserver with the name `rhcos_image_cache`. This pod serves the `bootstrapOSImage` image in the `install-config.yaml` file, `${RHCOS_PATH}${RHCOS_QEMU_URI}?sha256=${RHCOS_QEMU_SHA_UNCOMPRESSED}`, for deployment.  
```

Comment:
Can you put this sentence in a callout and remove the "above command" part? 
> <1> Creates a caching webserver with the name `rhcos_image_cache`. This pod serves the `bootstrapOSImage` image in the `install-config.yaml` file, `${RHCOS_PATH}${RHCOS_QEMU_URI}?sha256=${RHCOS_QEMU_SHA_UNCOMPRESSED}`,  for deployment. 

This is only my perspective so feel free to ignore, but the second sentence is hard to read because of the env variables are pushing the "for deployment" at the end of it, making it feel disconnected. If it makes sense, something like:
> <1> Creates a caching webserver with the name `rhcos_image_cache`. This pod serves the `bootstrapOSImage` image in the `install-config.yaml` file for deployment.

I'd try to add another sentence about the variables if they **have** to be included here.

Reply:
Updated as suggested.